commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -65,13 +65,24 @@
       converter = UInt64Converter.class)
   private UInt64 altairForkEpoch;
 
+  @CommandLine.Option(
+      names = {"--enable-key-manager-api"},
+      paramLabel = "<BOOL>",
+      description = "Whether to enable the key manager API to manage key stores (default: ${DEFAULT-VALUE}).",
```

Comment:
Perhaps simplify this to "Enable the key manager API to manage key stores"

Reply:
updated + fixed formatting